### PR TITLE
Update README as sd-driver is now part of mbed-os

### PR DESCRIPTION
**Pull request by LDong-Arm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,18 +34,6 @@ Move into the newly created directory:
 ```
 cd mbed-os-example-sd-driver
 ```
-	
-If the mbed-os library was not automatically added (see trace above), do the following to import mbed-os:
-
-```
-mbed new .
-```
```
Comment (LDong-Arm) [inline README.md:42]: I think this happens (if the user uses an up-to-date Mbed CLI). Other examples don't mention it at all.

**Comment by LDong-Arm** [comment]
@evedon @adbridge

**Comment by evedon** [comment]
Fixes #9

File: README.md (hunk 1 above)
Comment (evedon) [reply README.md:42]: It's should be fine now that the library is part of mbed-os

**Pull request merged**